### PR TITLE
[Merged by Bors] - Correct NiFi configuration overrides documentation.

### PR DESCRIPTION
**Pull request by Jimvin**
# Description

The documentation erroneously states that configuration overrides are not supported while also documenting how to do this. Corrected the error and added and example of adding a property to bootstrap.conf.

## Review Checklist

- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/ROOT/pages/usage.adoc [hunk 1]
```diff
@@ -85,8 +85,7 @@ The cluster definition also supports overriding configuration properties and env
 IMPORTANT: Do not override port numbers. This will lead to cluster malfunction.
 
 === Configuration Properties
-
-Currently, not supported for any file.
+You can override configuration properties by using configOverrides in your service configuration as demonstrated in the xref:config_properties.adoc[configuration properties] docs.
```
Comment (maltesander) [inline docs/modules/ROOT/pages/usage.adoc:88]: I looked at the code and its only working for bootstrap.conf and nifi.properties. Maybe its worth mentioning that?

Comment (Jimvin) [reply docs/modules/ROOT/pages/usage.adoc:88]: I've only given examples for overriding bootstrap.conf and nifi.properties in the docs, but being explicit about this limitation does make sense.

File: docs/modules/ROOT/pages/config_properties.adoc [hunk 2]
```diff
@@ -108,6 +108,8 @@ The following snippet shows an example of how this would look in the NifiCluster
           sensitivePropertyKeySecret: nifi-sensitive-property-
         configOverrides:
           nifi.properties:
-            aws.region: eu-west-1
+            nifi.config.property: eu-west-1
+        bootstrap.conf:
+            boostrap.config.property: some_string
```
Comment (sbernauer) [inline docs/modules/ROOT/pages/config_properties.adoc:113]: Is the indentation correct?

Comment (Jimvin) [reply docs/modules/ROOT/pages/config_properties.adoc]: Fixed

**Review by maltesander — APPROVED** [review]
LGTM.

**Pull request closed without merging**